connect/eaas/runner/main.py

```diff
@@ -60,6 +66,33 @@ def configure_logger(debug):
     )
 
 
+def check_runner_version(skip_check):
+    if skip_check:
+        return True
```

Comment:
The return value is consumed by the caller ? 
Otherwise you can just `return`

Reply:
By the tests